src/System.Runtime.Serialization.Json/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uapaot;
```

Comment:
Sure. I ran the command from source root. A few .sln and .csproj files were changed, should I checkin them?